src/content/docs/apm/agents/php-agent/other-integrations/magento-integration.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Magento monitoring integration
```

Comment:
The product name is 'Magento Business Insights'. There is already a Magento quickstart, so we don't want to duplicate it.